### PR TITLE
Added a formal ontology based on SKOS

### PR DESCRIPTION
**Pull request by proycon**
In order to allow repostatus.org to be used in the linked open data
world, it would help to have the concepts expressed in a way that allows
linking with the wider semantic web. This is my attempt at doing so in
an as unobtrusive way as possible. I opted for JSON-LD as the
serialisation format, which is easily parsable and even retains some
value even if you were ignore the context that maps it to an RDF model.

I defined the all of the concepts defined in repostatus as SKOS Concepts
(https://www.w3.org/TR/skos-reference/), as that is a common and
relatively simple way to organize knowledge. The concepts together form
a ConceptScheme. I added opengraph vocabulary to link to the badge image
(as SKOS has no vocabulary for that).

Ideally this ontology.jsonld file should be served on repostatus.org if
the client sends an HTTP request with header `Accept: application/json+ld` (content
negotation).

### CODE REVIEW

**Comment by jantman** [comment]
@proycon This looks good to me... I'm not really familiar with this stuff, but I don't really see any reason not to merge this and release it as 2.1.1.

Regarding serving the JSON via content negotiation, unfortunately, I don't think that's possible. repostatus.org is currently served from GitHub Pages. Unfortunately, I don't have any desire to set up any more involved hosting for this.

File: gh_pages/index.md [hunk 1]
```diff
@@ -78,6 +80,7 @@ Machine parsing is quite simple: first, search through any files in the top-leve
 
 There are some example parsers listed on the [parsers](/parsers) page.
 
+
```
Comment (waldyrious) [inline gh_pages/index.md:83]: Why was this line added?

**Comment by proycon** [comment]
> This looks good to me... I'm not really familiar with this stuff, but I don't really see any reason not to merge this and release it as 2.1.1.

Thanks!

> Regarding serving the JSON via content negotiation, unfortunately, I don't think that's possible. repostatus.org is currently served from GitHub Pages. Unfortunately, I don't have any desire to set up any more involved hosting for this.

I understand, that's ok, people can always still reference the JSON-LD file directory from their `@context`.

File: gh_pages/index.md (hunk 1 above)
Comment (proycon) [reply gh_pages/index.md]: Removed now

File: gh_pages/index.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ This is accomplished by including a simple badge or URL in your project's README
 * <a name="unsupported"></a>__Unsupported__ – The project has reached a stable, usable state but the author(s) have ceased all work on it. A new maintainer may be desired.
 * <a name="moved"></a>__Moved__ - The project has been moved to a new location, and the version at that location should be considered authoritative. This status should be accompanied by a new URL.
 
-These status descriptions and the URLs to the corresponding icons are also available in a [JSON file](/badges/latest/badges.json).
+These status descriptions and the URLs to the corresponding icons are also available in a [JSON file](/badges/latest/badges.json) or in a more formal ontology using SKOS, defined in a [JSON-LD file](/badges/latest/ontology.jsonld).
```
Comment (waldyrious) [inline gh_pages/index.md:22]: It might be useful to link "SKOS" to https://www.w3.org/TR/skos-reference/ to provide context.

Comment (proycon) [reply gh_pages/index.md]: Agreed! Added now in 1a1f87d454aad2cc2f85956304e53cd6fba3155d

**Comment by jantman** [comment]
@waldyrious Does this look ready to merge to you? If so, I can merge and release this tomorrow.

**Comment by waldyrious** [comment]
> @waldyrious Does this look ready to merge to you?

Absolutely :) Thanks for asking.

**Comment by proycon** [comment]
(friendly reminder: this is still awaiting merge)

**Comment by proycon** [comment]
may i give this another bump?

**Comment by jantman** [comment]
So sorry!

**Comment by jantman** [comment]
So sorry for letting this sit so long. Thank you for this! It's now live on the site.

**Comment by proycon** [comment]
No problem, thanks!
